core/blockchain.go

```diff
@@ -227,6 +236,7 @@ func ExecuteBlockEphemerallyForBSC(
 		return nil, fmt.Errorf("gas used by execution: %d, in header: %d", *usedGas, header.GasUsed)
 	}
 	if !vmConfig.NoReceipts {
+		bloom = newBlock.Bloom()
 		if newBlock.Bloom() != header.Bloom {
```

Comment:
statelessExec is used in other places in EBE...can you check those and ensure it's reflected here too